man/systemd.exec.xml

```diff
@@ -139,6 +139,14 @@
         not be able to log via the syslog or journal protocols to the host logging infrastructure, unless the
         relevant sockets are mounted from the host, specifically:</para>
 
+        <para>The host's
+        <citerefentry><refentrytitle>os-release</refentrytitle><manvolnum>5</manvolnum></citerefentry>
+        file will be made available for the service (read-only) as
+        <filename>/run/host/os-release</filename>.
+        It will be updated automatically on soft reboot (see:
+        <citerefentry><refentrytitle>systemd-soft-reboot.service</refentrytitle><manvolnum>8</manvolnum></citerefentry>),
+        in case the service is configured to survive it.</para>
```

Comment:
i am sorry, but this is not a good idea. This should be opt-in, not always done.

I don#t understand why this cannot just be done in portabled as before? why make this happen always? i think this should be reverted.

It's fine to leak this info into the sandbox in the more specific portabled usecase, but in the generic service namespace logic? yuck no. and there's no technical reason to require this. In fact you might as well just do this stuff as part of the profile logic now.